src/barrier-picker/SpreadBarrierCard.js

```diff
@@ -42,7 +42,7 @@ export default class SpreadBarrierCard extends Component {
                 <InputGroup
                     type="number"
                     label={`Amount per point (${currency})`}
-                    value={spreadInfo.amountPerPoint}
+                    defaultValue={spreadInfo.amountPerPoint}
```

Comment:
Using value would fix the value though and therefore close it for modification or doesn't it ? It it need to use value why not use label ? Or am i confusing something ?